### DISCUSSION TITLE
Update Banked XP to work with Giant's Foundry

### DISCUSSION THREAD

**Discussion by Shiroo49** [runelite/runelite#15977] [Q&A] [answered]
Currently there's no option on Banked XP to show the amount of exp you can get per bar at Giant's Foundry. This could possibly be done with averages since there is no definitive amount of exp you get per bar. However; there is a table on the wiki that displays information that could be used:

![image](https://user-images.githubusercontent.com/73411477/205095818-d5efaaaa-9d21-40c5-801d-f91ed4ca0a36.png)

The average bars used per hour and the average exp gained per hour using optimal bar combinations could be used to obtain a rough estimate of exp/bar

Please consider this, as it would be lovely to see in RL

**Comment by Nightfirecat** [comment, marked as answer]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
